README.md

```diff
@@ -70,6 +72,7 @@ Participation" depending on the conference.
 
 ### Proposing talks
 
+* [Finding Your Killer Talk Idea](http://ladiesintech.com/finding-your-killer-talk-idea/) by Rachel Nabors
```

Comment:
Could you please add this to the end of this section's list rather than the start?

(I should add it to the CONTRIBUTING guide; I'll do this momentarily).